build.yaml

```diff
@@ -1370,6 +1370,29 @@ steps:
       - merge_code
       - build_hail_jar_and_wheel
       - hail_ubuntu_image
+  - kind: buildImage2
+    name: hail-benchmark-image
+    dockerFile: /io/repo/benchmark/Dockerfile
+    contextPath: /io/repo
+    publishAs: hail-benchmark
+    buildArgs:
+      - name: HAIL_UBUNTU
+        value:
+          valueFrom: hail_ubuntu_image.image
+      - name: HAIL_WHEEL_DIR
+        value: /wheel
+    inputs:
+      - from: /repo/benchmark
+        to: /io/repo/benchmark
```

Comment:
While you're here, it would be great if you could throw the benchmark `pinned-requirements.txt` into the `hail_linting_image` step. That is intentionally a kitchen sink of all the pip requirements across the repo to ensure that they are always all compatible with each other (we don't pin two different versions of transitive dependencies in different places).

Reply:
Will do